certbot-apache/certbot_apache/tests/parsernode_util_test.py

```diff
@@ -48,8 +48,8 @@ def test_parsernode(self):
         params = self._setup_parsernode()
         ctrl = self._setup_parsernode()
 
-        ancestor, dirty, filepath, metadata = util.parsernode_kwargs(params)
-        self.assertEqual(ancestor, ctrl["ancestor"])
+        ancestors, dirty, filepath, metadata = util.parsernode_kwargs(params)
+        self.assertEqual(ancestors, ctrl["ancestors"])
```

Comment:
This change makes me think that a) maybe we should use a set here instead of a tuple, and b) we might need to fix the equality checks in `assertions`

Reply:
I think that's right-- we might want hashcodes for these objects anyhow since we essentially construct one in the `__eq__` functions.

Reply:
As we talked about, this is probably overly complex. We don't use `__eq__` outside of testing, so I changed the tests to use `assertions` instead and removed `__eq__` altogether. That way we don't have to check ancestor parity, as the `assertions` library already doesn't. If it turns out we need to, we can implement that in the future.